eventDataScript.js

```diff
@@ -0,0 +1,296 @@
+    
+    setCookie(cookieName,newText,750); //updates event data cookie
+    setCookie('currCheckedEvent','',750); //last checked event was deleted, restart cookie
+    
+    //delete team names for that event
+    var cname = '';
+    var cookieList = document.cookie.split(';');
+    for (var i = 0; i < cookieList.length;i++)
+    {
+        var currCname = cookieList[i].trim().split('=')[0];
+        if (currCname.endsWith(cnameEnding))
+        {
+            cname = currCname;
+        }
+    }
+    deleteCookie(cname);
+
+    document.location.reload();
+}
+
+
+function setTeamTable(season, event)
+{
+    var currEvent = getCurrEvent();
+    var currSeason = getCookie('currCheckedSeason');
+    if (currEvent !== ' ')
+    {
+        $('#teamTableTitle').text('Teams for event "' + currEvent + '."');
+    }
+    $("#eventSpecificTeamTable").html('<tr><th><b>Team Number</b></th> <th><b>Team Name</b></th> </tr>');
+
+    var teamsJSON = JSON.parse(getCookie(event_data_cookie_name(season,currEvent)));
+    if (teamsJSON == '{}') return;
+    const pairs = Object.entries(teamsJSON);
+    for (var i = 0; i < pairs.length; i++)
+    {
+        var teamNumber = pairs[i][0];
+        var teamName = pairs[i][1];
+
+        var newTRHtml = "<tr><td>" + teamNumber + 
+        "</td><td>" + teamName + "</td></tr>";
+
+        if ($('#eventSpecificTeamTable').html().indexOf(newTRHtml) < 0 &&
+        (teamNumber != '' || teamName != ''))
+        {
+            $('#eventSpecificTeamTable').append(newTRHtml);
+        }
+    }
+}
+
+//onclick for deleting a specific team based on number
+function deleteSpecificTeam()
+{
+    var input = $('#deleteSpecificTeamTextBox').val().trim(); //should be team number
+    if (input === '') return;
+
+    var season = getCookie('currCheckedSeason');
+    var currEvent = getCurrEvent(); 
+    var cname = event_data_cookie_name(season,currEvent);
+    var teamsJSON = JSON.parse(getCookie(cname));
+
+    delete teamsJSON[input];
+    setCookie(cname,JSON.stringify(teamsJSON),750);
+    setTeamTable(season,currEvent);
+    $("#deleteSpecificTeamTextBox").val("").focus();
+}
+
+$(document).ready(function(){
+    loadPage();
+
+    const Enter_key_code = 13;
+    $('#newEventInputBox').keypress(function(e){
+        if (e.keyCode == Enter_key_code) addNewEvent();
+    });
+
+    //event specific teams code
+    $('#teamName').keypress(function(e){
+        if (e.keyCode == Enter_key_code) $('#teamSaveButton').click();
+    });
+
+    $('#deleteSpecificTeamTextBox').keypress(function(e){
+        if (e.keyCode == Enter_key_code) deleteSpecificTeam();
+    });
+
+    $('#teamSaveButton').click(function(){
+        var currSeason = getCookie('currCheckedSeason');
+        var currEvent = getCurrEvent();
+        var teamNumber = parseInt($("#teamNumber").val().trim());
+        var teamName = $("#teamName").val().trim();
+        var teams = {};
+
+        if (teamNumber === "" || teamName === "")
```

Comment:
this is a pretty long function, go ahead and put it outside loadPage and use the name here:

```js
function saveTeam() {
...
}
...
   $('#teamSaveButton').click(saveTeam);
...
```